packages/javascript-sdk/src/index.ts

```diff
@@ -103,17 +103,39 @@ function initializeNamespacedClient(namespace: string, client: UsermavenClient)
         while (queue.length > 0) {
             const item = queue.shift();
             if (item) {
-                (window as any)[namespace].apply(null, item);
```

Comment:
@main03 Did we remove the namespace code? Can we check if this works when we are using namespaces?